go.mod

```diff
@@ -3,17 +3,16 @@ module github.com/palomachain/paloma
 go 1.19
 
 require (
-	github.com/CosmWasm/wasmd v0.30.0
+	github.com/CosmWasm/wasmd v0.11.1
 	github.com/CosmWasm/wasmvm v1.1.1
-	github.com/cosmos/cosmos-sdk v0.45.11
-	github.com/cosmos/ibc-go/v4 v4.2.0
+	github.com/cosmos/cosmos-sdk v0.39.3
```

Comment:
Do not understand why this PR actually downgrades CosmosSDK version. Also see the message in `go mod tidy`-ing:
```
go: github.com/palomachain/paloma/app imports
        github.com/cosmos/cosmos-sdk/x/evidence/keeper: github.com/cosmos/cosmos-sdk/x/evidence@v0.0.0-20230228204446-44495e7a7934: parsing go.mod:
        module declares its path as: cosmossdk.io/x/evidence
                but was required as: github.com/cosmos/cosmos-sdk/x/evidence
```
meaning that the structure of CosmosSDK is different from the current version